src/bots/depositor.py

```diff
@@ -274,3 +275,57 @@ def _send_deposit_tx(
             self._flashbots_works = True
 
         return success
+
+    def _prepare_transaction(
+        self,
+        block_number: int,
+        block_hash: Hash32,
+        deposit_root: Hash32,
+        staking_module_id: int,
+        staking_module_nonce: int,
+        payload: bytes,
+        guardian_signs: tuple[tuple[str, str], ...]
+    ) -> ContractFunction:
+        """
+        It either follows a regular flow or builds a direct deposit transaction.
+
+        Conditions to build direct deposit transaction are:
+        1. Env variable is set
+        2. balance in the vault is not 0
```

Comment:
Let's move it to variable and set 1 ETH by default 
Like MAX_GAS_FEE env

Reply:
Done